data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -2,12 +2,21 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 35,
    "id": "71c4acd0",
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
This is yet another undocumented convention, I'm not sure where the best place to write this down would be, but we've agreed to "clear output" from our python notebooks before committing changes to git to cleanup the git diff.

Reply:
Did you mean to resolve this? Would be great to clear these outputs before we merge. Thanks!